src/libraries/System.Formats.Cbor/src/System.Formats.Cbor.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>true</IsPackable>
+    <UseIntellisenseDocumentationFile>true</UseIntellisenseDocumentationFile>
```

Comment:
These 3 csproj files are supposed to have their documentation source of truth already in triple slash comments. But in a future PR, I will do an additional pass with api-docs-sync to ensure the docs are fully synced.

I tested by modifying a triple slash comment of a public API, and I could confirm that the final file in artifacts/bin/docs contained the change.